README.md

```diff
@@ -158,4 +158,22 @@ Once the Pull Request is created, manual action should be perform to:
 - If no changes are detected, the workflow will silently skip the bump.
 
 
+## Debugging job (Post-Sonatype Migration)
```

Comment:
As we migrated, should we still use this title ? I suggest to use " Snapshot & debug release job"